docs/generated/sql/bnf/alter_role_or_user_stmt.bnf

```diff
@@ -1,3 +1,3 @@
 alter_role_or_user_stmt ::=
```

Comment:
This can be deleted - I forgot to delete this auto generated file after changing this syntax.